.github/workflows/tag_image_push.yml

```diff
@@ -142,15 +145,27 @@ jobs:
           TAGS_TO_PUSH=$(get_tags_to_push ${{ env.RELEASE_VERSION }})
           echo "TAGS_TO_PUSH=$TAGS_TO_PUSH"
           TAGS_ARG=""
+          DEFAULT_JDK=11  # Set the default JDK version
           for tag in ${TAGS_TO_PUSH[@]}
           do
-             TAGS_ARG="${TAGS_ARG} --tag ${{ env.DOCKER_ORG }}/hazelcast:${tag}${{ matrix.suffix }}"
+            # Add a tag without JDK version for default JDK
```

Comment:
Use jdk variable in `Store docker logs as artifact` and `Get docker logs`

Reply:
done